vars/archiveArtifacts.groovy

```diff
@@ -0,0 +1,25 @@
+// vars/archiveArtifacts.groovy
+
+/**
+ * run.groovy
+ *
+ * Wrapper for archiveArtifacts step to allow for skipped tests.
+ *
+ */
+
+def call(Map config = [:]) {
+    println "Entering archiveArtifacts override."
+    def script = '''if [ "${NO_CI_TESTING}" == 'true' ]; then
```

Comment:
Where/how is `NO_CI_TESTING` defined?

Reply:
It is defined as a Jenkins-Wide environment variable.